bin/package-electron

```diff
@@ -36,6 +36,7 @@ fi
 
 yarn workspace loot-core build:node
 
+yarn workspace @actual-app/web generate:i18n
```

Comment:
Should this reuse the code in package-browser that filters down to usable translations?

https://github.com/actualbudget/actual/blob/15298703ac3f33b931b6a6dcea21c4c308f95e90/bin/package-browser#L7-L14